src/transformStateWithClones.js

```diff
@@ -7,7 +7,42 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const result = [];
+  let stateCopy = { ...state };
+
+  for (let i = 0; i < actions.length; i++) {
+    let actionsType;
```

Comment:
Same as in previous task

Better to use this object property instead of variable
(you didn't use it outside switch/case, you store different values in it)
It's harder to read your code with it